src/views/portal/Portal.tsx

```diff
@@ -1,16 +1,16 @@
 import Head from "next/head";
-import { Container } from "react-bootstrap";
-
 import Router from "next/router";
 import { useContext, useEffect } from "react";
 
 import { TitleBanner } from "components";
+import { Container } from "react-bootstrap";
```

Comment:
Chore: When [importing components with React Bootstrap](https://react-bootstrap.github.io/getting-started/introduction#importing-components), we should import individual components. Also, we should try and keep the import ordering consistent, i.e. third-party in a group before first-party.